bundles/org.openhab.binding.volvooncall/src/main/java/org/openhab/binding/volvooncall/internal/dto/HvBattery.java

```diff
@@ -27,8 +27,17 @@
 public class HvBattery {
     public int hvBatteryLevel = UNDEFINED;
     public int distanceToHVBatteryEmpty = UNDEFINED;
+    /**
```

Comment:
I wonder if '`hvBatteryChargeStatusDerived`' should not rather be an `enum`

Reply:
It _could_ be, but we don't know the full range of values (We've seen many, but there could be more), and the API is subject to change by an external party, so I'd play it safe an continue with the String.

Reply:
Ok, keep it like this then.